stanford/bin/cut-release-branch

```diff
@@ -13,7 +13,9 @@ done
 : ${BRANCH_TO_CUT:=rc}
 : ${WORKSPACE:=${HOME}/workspace/cut_branch}
 : ${EMAIL_RECIPIENT:=''}
-date=$(echo "${BUILD_ID}" | head -c10)
+: ${EMAIL_SENDER:= 'no_reply@lagunita.stanford.edu'}
```

Comment:
Can we make the domain name be `build.lagunita.stanford.edu`
to reflect the machine/service from which it came?

I find this makes my email search/management a bit easier.

Reply:
And can we make the left-hand-side be `noreply` without punctuation?
That seems to be our most prevalent convention used elsewhere.

The underscore version isn't used anywhere else.

Reply:
and just because some shells can be finicky sometimes,
can we remove the space between `:=` and the quotes,
as done elsewhere.